src/features/Applet/Schedule/Calendar/MonthView/MonthView.tsx

```diff
@@ -1,16 +1,37 @@
 import { useEffect, useRef } from 'react';
 import CalendarMonthView from 'react-big-calendar/lib/Month';
 
+import { COEFFICIENT_SHOW_ONE_MORE_EVENT } from './MonthView.const';
+
 export type MonthViewType = typeof CalendarMonthView;
 
 export const MonthView = (props: MonthViewType) => {
   const monthRef = useRef<MonthViewType>();
+  const { date, activeView } = props.components;
 
   useEffect(() => {
-    if (monthRef.current?.state?.rowLimit < 3 || monthRef.current?.state.needLimitMeasure) {
-      monthRef.current.setState({ rowLimit: 3 });
-    }
-  });
+    const timeout = setTimeout(() => {
```

Comment:
do you really need this timeout?

Reply:
yep, unfortunately without timeout I have no access to container elements, the are undefined